RelhaxModpack/RelhaxModpack/Shortcuts/ShortcutUtils.cs

```diff
@@ -106,6 +125,70 @@ internal interface IShellLink
             void Resolve(IntPtr hwnd, int fFlags);
             void SetPath([MarshalAs(UnmanagedType.LPWStr)] string pszFile);
         }
+
+        // needed for CreateShortcut
+        [Flags]
+        public enum StgmConstants
```

Comment:
REEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEE

Reply:
These are specific to that one class and they're in a `#region` of "gross shortcut stuff". I could justify leaving them there?

Reply:
var justified = true